content/blog/computer-ergonomics/index.md

```diff
@@ -0,0 +1,52 @@
+
+Developers sit in front of a computer for hours on end. As you may already know, sitting is not the most natural position for humans to be in for a prolonged period of time. Small repetitive strain on the body from bad posture can cause people to experience muscle strain and headaches.
+
+![Back pain](./back-pain.jpg)
+
+Being aware and mindful of the body position you are in during work can help reduce or completely mitigate bad posture habits. On the contrary, just getting a shiny new expensive standing desk will not solve all your problems.
+
+### Computer ergonomics
+
+The general principle of ergonomics is to keep the body in a neutral position and avoid slouching over. Take time to set up the work environment, so you can feel better and be more productive later.
+
+![Sit and stand posture](./sit-stand-posture.jpg)
+
+- **Computer screen**. Laptop screens are not ergonomically designed products, because the screen is too low and we lean in towards it. Having an external monitor about an arm's length away is the first step towards a better workplace. Eyes should be aligned about 5 cm (2 inches) below monitor height.
+- **Chair**. Try different chairs. Generally, it should have good lumbar support, be taller than your neck and adjustable. Modify it according to the height of the table, so your elbows are at a 90-degree angle. Use a footrest if your feet don't reach the floor.
+- **Breaks**. Our bodies are built to move, not to sit. Take regular breaks. It's recommended to do it about every 30 minutes.
+- **Wrists**. Using palms for support when typing can reduce risks of damaging the wrist.
+- **Light**. Having sufficient light when working with a computer can help reduce eye strain.
+
+### Standing Desks
+
+Sit-stand workstations are becoming really popular, especially in the tech world. Completely switching to standing all day will not be much better than sitting. It's important to change the position often, try to do it every hour or so. There has been a study concluding that people who stand more at work tend to spend less time sitting at home, living a more active lifestyle.
+
+> In 2015, Denmark passed a law making sit-stand desks a legal requirement for the employees who work more than 30h per week. This is why Danish design has been prevalent in the sit-stand desk industry.
+
+### Equipment
+
+- Mouse. If you have large hands, then choose a bigger mouse. There are also some specially designed ergonomic mouses like [Logitech MX Vertical](https://www.logitech.com/en-us/product/mx-vertical-ergonomic-mouse).
+- Apps. There are apps like Stretchly that remind you to take breaks. Find more about taking breaks in [previous article](https://devhealth.io/breaks/).
+
+### Conclusion
+
+Invest in the workspace or ask employer to provide you with the best possible equipment. Taking good care of your body by minimizing the stress will help increase productivity in the long run.
```

Comment:
_by minimizing the stress_

this sentence would probably go better together with the next paragraph since i'm not sure the stress goes together with the equipment from the previous sentence.